js/tips-tricks.md

```diff
@@ -0,0 +1,26 @@
+# JavaScript / Tips & Tricks
+
+#### Singleton classes
+
+```js
+class Foo {
```

Comment:
Let's use a real example we have elsewhere:
```suggestion
import ElectronStore from "electron-store";
```

Reply:
https://github.com/c-hive/guides/pull/43/commits/8fed5a0eedb9b72041c7049afcd2709a5f4354a6